rc-events-service/src/main/scala/com/epam/random_coffee/events/model/Event.scala

```diff
@@ -1,5 +1,20 @@
 package com.epam.random_coffee.events.model
 
-final case class Event(id: EventId, name: String)
+import java.time.Instant
 
 final case class EventId(value: String) extends AnyVal
+
+final case class Event(id: EventId, name: String)
+
+final case class Author(id: String) extends AnyVal
+
+final case class DateForEvent(date: Instant) extends AnyVal
```

Comment:
Don't see any reason, why we should wrapp Instant.
Mb I'm wrong. @myazinn what do you think?
I mean, there is no additional logic for validation or additional methods, but it leads us to add implicits to repository for no reason

Reply:
Thank you for this note. 
Wrapper class for Instant and related code has been deleted.

Reply:
@GrigoriiBerezin agree
It's a thin line, but this one looked over engineered